index.js

```diff
@@ -1320,7 +1355,8 @@
   //. const NonZeroInteger = $.NullaryType(
   //.   'my-package/NonZeroInteger',
   //.   'http://example.com/my-package#NonZeroInteger',
-  //.   x => $.test([], Integer, x) && x !== 0
+  //.   Integer,
```

Comment:
This is referring to the Integer type defined above. One day we should write a new example which does not include a redefinition of a type provided by sanctuary-def.